app/api/api_v1/routers/organisation.py

```diff
@@ -0,0 +1,57 @@
+import logging
+
+from fastapi import APIRouter, HTTPException, status
+
+from app.errors import RepositoryError, ValidationError
+from app.model.organisation import OrganisationReadDTO
+from app.service import organisation as organisation_service
+
+organisations_router = APIRouter()
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@organisations_router.get(
+    "/organisations",
+    response_model=list[OrganisationReadDTO],
+)
+async def get_all_organisations() -> list[OrganisationReadDTO]:
+    """Retrieve all organisations.
+
+    :raises HTTPException: If the corpus type is not found.
+    :return CorpusTypeReadDTO: The requested corpus type.
+    """
+    try:
+        return organisation_service.all()
+    except RepositoryError as e:
+        raise HTTPException(
+            status_code=status.HTTP_503_SERVICE_UNAVAILABLE, detail=e.message
+        )
+
+
+@organisations_router.get(
+    "/organisations/{organisation}",
```

Comment:
```suggestion
    "/organisations/{organisation_id}",
```

nit: following docs - maybe `_id` is more explicit https://fastapi.tiangolo.com/tutorial/path-params/

Reply:
Done